app/views/articles/show.html.erb

```diff
@@ -103,73 +103,64 @@
-                <span class="crayons-logo crayons-logo--l mr-3">
-                  <img src="<%= Images::Profile.call(@organization.profile_image_url, length: 50) %>" class="crayons-logo__image" alt="<%= @organization.name %> profile image">
-                </span>
-                <%= @organization.name %>
-              </a>
-            <% end %>
-            <h1 class="fs-3xl m:fs-4xl l:fs-5xl fw-bold s:fw-heavy lh-tight mb-4 <%= @article.title_length_classification %>">
+            <div class="flex s:items-start flex-col s:flex-row">
+              <div id="action-space" class="crayons-article__actions mb-4 s:mb-0 s:order-last"></div>
+              <div class="flex flex-1 mb-5 items-start">
+                <div class="relative">
+                  <% if @organization %>
+                    <a href="<%= @organization.path %>"><img src="<%= Images::Profile.call(@organization.profile_image_url, length: 50) %>" class="radius-default align-middle" width="40" height="40" alt="<%= @organization.name %> profile image"></a>
+                    <a href="/<%= @user.username %>" class="absolute -right-2 -bottom-2 radius-full border border-solid border-2 border-base-inverted inline-flex">
+                      <img class="radius-full align-middle" src="<%= Images::Profile.call(@user.profile_image_url, length: 50) %>" width="24" height="24" alt="<%= @user.name %>" />
+                    </a>
+                  <% else %>
+                    <a href="/<%= @user.username %>"><img class="radius-full align-middle" src="<%= Images::Profile.call(@user.profile_image_url, length: 50) %>" width="40" height="40" alt="<%= @user.name %>" /></a>
+                  <% end %>
+                </div>
+                <div class="pl-3 flex-1">
+                  <a href="/<%= @user.username %>" class="crayons-link fw-bold"><%= @user.name %></a>
+                  <% if @organization %>
+                    <span class="color-base-60">for</span> <a href="<%= @organization.path %>" class="crayons-link"><%= @organization.name %></a>
+                  <% end %>
+                  <p class="fs-xs color-base-60">
+                    <% if @article.published_timestamp.present? %>
+                      Posted on
+                      <%= local_date(@article.published_timestamp, show_year: @article.displayable_published_at.year != Time.current.year) %>
+                    <% end %>
+                    <% if @article.co_author_ids.present? %>
+                      with <%= @article.co_author_name_and_path.html_safe %>
+                    <% end %>
+
+                    <% if should_show_updated_on?(@article) %>
+                      &bull; Updated on <%= local_date(@article.edited_at, show_year: @article.edited_at.year != Time.current.year) %>
+                    <% end %>
+
+                    <% if should_show_crossposted_on?(@article) %>
+                      &bull; Originally published at
```

Comment:
It's the changes to the markup in this area that's causing the Cypress test to fail - Cypress unfortunately isn't great at matching text that's broken up by different html chunks

If you change line 72 in https://github.com/forem/forem/blob/4858b75bb58b130f3257689d480bed734231c06b/cypress/integration/seededFlows/publishingFlows/postOptions.spec.js#L72

to:

`cy.findByText(/Originally published at/).should('exist');`

It will look for a partial match instead and start passing again 🚀